packages/components/psammead-media-player/README.md

```diff
@@ -35,6 +37,7 @@ Assuming `showPlaceholder` is `true`, the `placeholderSrc` will be what image to
 | Argument  | Type                | Required | Default | Example         |
 |-----------|---------------------|----------|---------|-----------------|
 | `src` | string | Yes   | - | `http://foobar.com/embeddable_endpoint` |
+| `placeholderSrcset` | string | Yes   | - | `http://foobar.com/images/ic/512xn/p01k6mtv.jpg` |
```

Comment:
`512xn` is not a valid rez format
> Note: if you specify an incorrect/unavailable resolution you will get the following error:
> `You don't have permission to access images/ic/512xn/p01k6mtv.jpg`

format is usually `WxH` eg `512x512`, `512x288`, would be nice having the readme and test have instances of valid formats so they're clear enough

Reply:
Just followed the example given here https://github.com/bbc/simorgh/blob/ef0a03cd3949337912ff1e0a751da925a9549eaf/src/app/containers/MediaPlayer/helpers/placeholder.test.js#L7 as this will also be applied in simorgh